dev/NavigationView/NavigationView.cpp

```diff
@@ -1779,10 +1779,101 @@ void NavigationView::OnNavigationViewItemKeyDown(const winrt::IInspectable& send
     }
 }
 
+void NavigationView::ArrowKeyNavigationPolyfill(const winrt::NavigationViewItem& nvi, const winrt::KeyRoutedEventArgs& args)
+{
+    switch (args.Key())
+    {
+    case winrt::VirtualKey::Down:
+        if (auto nextFocusibleElement = FindNextFocusableElement(nvi, Direction::Down))
+        {
+            if (SetFocus(nextFocusibleElement, winrt::FocusState::Keyboard))
+            {
+                args.Handled(true);
+            }
+        }
+        break;
+    case winrt::VirtualKey::Up:
+        if (auto nextFocusibleElement = FindNextFocusableElement(nvi, Direction::Up))
+        {
+            if (SetFocus(nextFocusibleElement, winrt::FocusState::Keyboard))
```

Comment:
>if (SetFocus(nextFocusibleElement, winrt::FocusState::Keyboard)) [](start = 12, length = 64)

We talked about this offline, but I think this code does not handle the case where this element is not focusable correctly.